src/test/java/fi/otavanopisto/kuntaapi/test/server/integration/gtfs/ScheduleTestsIT.java

```diff
@@ -0,0 +1,132 @@
+      .contentType(ContentType.JSON)
+      .get("/organizations/{organizationId}/transportSchedules", getOrganizationId(0))
+      .then()
+      .assertThat()
+      .statusCode(200)
+      .body("id.size()", is(17))
+      .body("id[1]", notNullValue())
+      .body("name[1]", is("L+ Mikkeli"))
+      .body("days[1].size()", is(1))
+      .body("days[1][0]", is(6))
+      .body("exceptions[1].size()", is(10))
+      .body("exceptions[1][1].type", is("REMOVE"))
+      .body("exceptions[1][1].date",sameInstant(getInstant(2016, 7, 30, 0, 0, TIMEZONE_ID)))
+      .body("startDate[1]", sameInstant(getInstant(2016, 5, 1, 0, 0, TIMEZONE_ID)))
+      .body("endDate[1]", sameInstant(getInstant(2024, 4, 30, 0, 0, TIMEZONE_ID)));
+  }
+  
+  @Test
+  public void testFindSchedule() {
+    String organizationId = getOrganizationId(0);
+    given() 
+      .baseUri(getApiBasePath())
+      .contentType(ContentType.JSON)
+      .get("/organizations/{organizationId}/transportSchedules/{scheduleId}", organizationId, getOrganizationScheduleId(organizationId, 10))
+      .then()
+      .assertThat()
+      .statusCode(200)
+      .body("id", notNullValue())
+      .body("name", is("Ma+++"))
+      .body("days.size()", is(1))
+      .body("days[0]", is(1))
+      .body("exceptions.size()", is(22))
+      .body("exceptions[1].type", is("REMOVE"))
+      .body("exceptions[1].date",sameInstant(getInstant(2017, 1, 23, 0, 0, TIMEZONE_ID)))
+      .body("startDate", sameInstant(getInstant(2016, 5, 1, 0, 0, TIMEZONE_ID)))
+      .body("endDate", sameInstant(getInstant(2024, 4, 30, 0, 0, TIMEZONE_ID)));
+  }
+  
+  @Test
+  public void testOrganizationSchedulesNotFound() throws InterruptedException {
+    String organizationId = getOrganizationId(0);
+    String incorrectOrganizationId = getOrganizationId(1);
+    String organizationScheduleId = getOrganizationScheduleId(organizationId, 0);
+    
+    String[] malformedIds = new String[] {"evil", "*", "/", "1", "-1", "~"};
+    assertFound(String.format("/organizations/%s/transportSchedules/%s", organizationId, organizationScheduleId));
+    assertEquals(17, countApiList(String.format("/organizations/%s/transportSchedules", organizationId)));
+    
+    for (String malformedId : malformedIds) {
+      assertNotFound(String.format("/organizations/%s/transportSchedules/%s", organizationId, malformedId));
+    }
+    
+    assertNotFound(String.format("/organizations/%s/transportSchedules/%s", incorrectOrganizationId, organizationScheduleId));
+    assertEquals(0, countApiList(String.format("/organizations/%s/transportSchedules", incorrectOrganizationId)));
+  }
+
+  private void createPtvSettings() {
+    insertSystemSetting(PtvConsts.SYSTEM_SETTING_BASEURL, String.format("%s%s", getWireMockBasePath(), BASE_URL));
+    flushCache();
+  }
+  
+  private void deletePtvSettings() {
+    deleteSystemSetting(PtvConsts.SYSTEM_SETTING_BASEURL);
+  }
+  
+  private void createGtfsSettings(String organizationId) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused private "createGtfsSettings" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)